vendor.conf

```diff
@@ -1,5 +1,5 @@
-github.com/moby/buildkit 621596d36bbef2a0ffa1414ab1b679b4bca9e44d
-github.com/tonistiigi/fsutil c94d7a34379da5d1b7ac363527f049b586cd1916
+github.com/moby/buildkit 277037a77f32b689df342c2ac7ea1c093be4b52e
```

Comment:
@tonistiigi is this for README and moby's vendor.conf ?

Reply:
for no reason. just noticed it was old as well